src/text_file_functions.cpp

```diff
@@ -715,3 +715,84 @@ QString AOApplication::get_casing_can_host_cases()
   QString result = configini->value("casing_can_host_cases", "Turnabout Check Your Settings").value<QString>();
   return result;
 }
+
+int AOApplication::get_saved_ooc_format()
+{
+  QString value_string = configini->value("saved_ooc_format", "Plain text (default)").value<QString>();
+  format_ooc result;
+  if (value_string == "Raw HTML")
+    result = HTML;
+  #if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+    else if (value_string == "Markdown")
+      result = MARKDOWN;
+  #endif
+  else
+    result = PLAINTEXT;
+  return result;
+}
+
+Courtroom::save_log_result Courtroom::save_chatlog(bool is_ooc)
+{
+    QString log_type;
+    if (!is_ooc)
+      log_type = "IC";
+    else
+      log_type = "OOC";
```

Comment:
In this case, ternary does help with readability.
```suggestion
    QString log_type = is_ooc ? "OOC" : "IC";
```